modules/file/file.go

```diff
@@ -1032,6 +1056,60 @@ func (r *run) evaluateFile(file string) (err error) {
 	return
 }
 
+/* wantThis() implements boolean logic to decide if a given check should be a match or not
```

Comment:
wondering if this could be done with bitwise operators for clarity (wantThis). Not sure what is clearer to most people though.